object_tracker/match.py

```diff
@@ -4,35 +4,40 @@
 import numpy as np
 from typing import List
 from numpy import ndarray
-
+from omegaconf import DictConfig
 Roi = collections.namedtuple("Roi", ["min_x", "min_y", "w", "h"])
 
+class Matcher_Params:
+    def __init__(self, cfg: DictConfig):
+        self.num_orb_features = cfg.num_orb_features
+        self.bidirectional_enable = cfg.bidirectional_enable
+        self.bidirectional_thresh = cfg.bidirectional_thresh
+        self.min_points_for_find_homography = cfg.min_points_for_find_homography
+        self.detection_params = dict(
+            maxCorners=cfg.detection_params.maxCorners,
+            qualityLevel=cfg.detection_params.qualityLevel,
+            minDistance=cfg.detection_params.minDistance,
+            blockSize=cfg.detection_params.blockSize
+        )
+        self.tracking_params = dict(
+            winSize=cfg.tracking_params.winSize,
+            maxLevel=cfg.tracking_params.maxLevel,
+            criteria=(
+            cfg.criteria_params.CRITERIA_EPS | cfg.criteria_params.CRITERIA_COUNT,
+                cfg.criteria_params.criteria_param, cfg.criteria_params.criteria_param2),
+            minEigThreshold=cfg.tracking_params.minEigThreshold
+        )
 
 class Matcher:
```

Comment:
Please add documentation, When do we use it?

Reply:
what?

Reply:
Matcher